example_unique_job_test.go

```diff
@@ -14,44 +14,47 @@ import (
 	"github.com/riverqueue/river/riverdriver/riverpgxv5"
 )
 
-// Account represents a minimal account containing a unique identifier, recent
-// expenditures, and a remaining total.
+// Account represents a minimal account recent expenditures and remaining total.
```

Comment:
typo or missing words in here I think? "account recent expenditures"

Reply:
Ah dammit. Last second change and of course I messed it up. Thanks, fixed.